content/pages/index.md

```diff
@@ -5,34 +5,42 @@ save_as: index.html
 template: landing
 ---
 
-Welcome to the website of EuroSciPy!
-The next edition of EuroSciPy is [EuroSciPy 2023](/2023).
+# Welcome to the EuroSciPy conference
 
-News come on twitter: [@EuroSciPy](https://twitter.com/EuroSciPy).
+### The next edition is [EuroSciPy 2023](/2023/)
+
+#### [Schedule](/2023/schedule.html)
+
+#### [Tickets](/2023/tickets.html)
+
+News come on Twitter: [@EuroSciPy](https://twitter.com/EuroSciPy).
```

Comment:
I would not do that. I would only provide the index of the next website.

Reply:
We can revert if it will not work, but I believe, this is the fastest way to change the sections google display when you search for EuroSciPy, now half of them are pointing out to previous year, two months ago all of them were pointing out to previous year or even 2020/2021 (!)